crates/cargo-test-support/src/lib.rs

```diff
@@ -1359,3 +1359,86 @@ pub fn symlink_supported() -> bool {
 pub fn no_such_file_err_msg() -> String {
     std::io::Error::from_raw_os_error(2).to_string()
 }
+
+pub fn init_registry() {
+    crate::registry::init();
+    add_registry_packages(false);
+}
+
+pub fn init_alt_registry() {
+    crate::registry::alt_init();
+    add_registry_packages(true);
+}
+
+fn add_registry_packages(alt: bool) {
+    for name in [
```

Comment:
This is all specific to the add tests and shouldn't be in a general place.  It can confuse people on what they should use and if they use it, it couples the tests together making them harder to evolve since there are not shared requirements.

Reply:
I'll move them back and try and get them from `cargo_add`